src/EncompassRest/Loans/Enums/FreddieRelatedLoanLienPosition.cs

```diff
@@ -0,0 +1,14 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum FreddieRelatedLoanLienPosition
+    {
+        [EnumMember(Value = "Second Lien")]
```

Comment:
string values to match numeric values?( second = 2, third = 3, forth = 4)

Reply:
In case you didn't see I've done as you suggested.